kubernetes/config-maps/nginx-storage/default.conf.j2

```diff
@@ -207,14 +208,34 @@ server {
 
             os.rename("/srv/storage/" .. tmp_file_name, "/srv/storage/" .. header_path_with_filename)
             os.remove("/srv/storage/" .. tmp_file_name .. ".shactx")
+
+            -- Send confirmation to the conductor after uploaded file.
+            local http = require "resty.http"
+            local httpc = http.new()
+            local res, err = httpc:request_uri("{{ upload_report_url }}" .. header_path_with_filename, {
+            method = "POST",
+            })
+
+            if (res.status < 200 or res.status >= 300) then
+                ngx.status = 502
+                error_response = '{"error_message":"Failed to report upload. Conductor responded with status {' .. res.status .. '}", "conductor_message":"' .. res.body .. '"}'
```

Comment:
Please include `error_code` in the response.

You could also split it into multiple lines for readability.

Reply:
Done.